src/WebJobs.Script.WebHost/WebScriptHostBuilderExtension.cs

```diff
@@ -37,7 +38,7 @@ public static class WebScriptHostBuilderExtension
                     services.ConfigureOptions<CustomHttpHeadersOptionsSetup>();
                     services.ConfigureOptions<HostHstsOptionsSetup>();
                 })
-                .AddScriptHost(webHostOptions, configLoggerFactory, webJobsBuilder =>
+                .AddScriptHost(webHostOptions, metadataProvider, configLoggerFactory, webJobsBuilder =>
```

Comment:
As discussed, we should be able to refactor this so we're not modifying these extension methods to take additional services.